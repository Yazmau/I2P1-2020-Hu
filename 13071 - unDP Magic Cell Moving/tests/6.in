1000
1 1 640409427
3 386727211
3 232566849
1 1 292309906
1 3 604953294
4
4
2 1
3 121903903
3 414463421
4
2 3
2 1
4
2 3
1 3 924541501
3 748701061
1 0 765076369
1 3 868425023
3 392742323
2 1
1 3 824658520
4
4
1 1 456751806
1 0 566809202
1 0 566312336
1 2 562888302
4
4
3 261330565
4
4
4
1 2 678005146
2 3
1 0 680816425
2 0
2 2
3 539041769
4
1 2 353192088
4
1 2 983238630
2 3
2 1
1 1 410758194
3 646399857
1 1 308544045
2 0
1 3 197944943
2 3
3 788758322
2 1
4
4
2 2
2 0
2 3
2 1
4
4
1 0 709763771
4
4
2 2
4
3 603740891
2 1
4
1 1 691821885
4
4
1 3 370689501
1 1 560616637
2 2
3 859011833
2 0
2 3
1 2 833781429
1 1 536887923
4
3 152632316
4
1 2 597811775
4
4
2 3
4
2 1
1 1 920383236
2 1
2 0
1 0 379307831
3 722760997
1 0 325541928
3 806359821
4
4
4
2 2
4
4
1 1 358377101
3 262685656
1 1 63771355
3 528042240
1 3 890615869
4
1 1 871026988
2 3
2 2
4
2 0
2 1
3 649471902
2 0
4
2 2
4
1 1 381287085
2 3
3 784498461
2 2
3 647910330
3 534342347
3 851169531
2 0
3 439256833
1 2 835316709
3 750855735
2 3
4
4
4
1 3 843882482
3 292016870
3 467250912
1 0 314356627
1 3 997192129
4
4
3 289738367
4
2 3
2 1
3 807613404
4
4
1 3 417222325
2 1
2 0
3 340768348
2 3
4
1 2 424008614
2 2
4
1 1 646722049
3 941755784
3 459977314
3 557419813
1 0 360478700
1 0 254525128
2 1
4
4
3 69876497
1 3 226740341
4
1 2 581441878
2 2
3 334684178
1 3 671974976
2 0
1 0 66893810
2 3
4
3 895326369
1 0 886053366
2 0
3 131218052
1 3 339190501
1 1 944890862
1 0 373943673
3 981930636
4
2 3
3 325238052
3 516925237
4
4
1 0 483954289
3 755896286
3 461268615
2 0
3 654418045
1 2 863567294
3 879215617
3 700703916
3 381678280
2 2
1 1 257217755
2 1
1 0 701671915
3 900863891
1 0 460991427
1 3 272842311
2 3
3 770564408
3 833311783
4
2 1
1 2 548896558
4
2 3
4
4
2 1
4
2 2
3 775835646
2 0
2 1
1 2 593570905
3 749062786
2 2
3 178630778
2 0
4
3 593096016
4
1 2 30305080
3 183130070
1 3 833489073
2 0
4
3 908081332
2 2
2 2
1 1 346939706
1 0 288827700
1 3 353467132
2 3
1 3 621861047
4
3 511147072
3 247904116
3 287133520
1 3 199301257
3 384564454
2 3
1 2 968170837
2 1
3 245750888
1 0 580211560
4
3 201046694
1 2 118210004
4
3 476388253
3 741311132
3 894420438
4
1 3 318372840
1 0 864060032
2 3
4
1 3 981522897
3 917958945
4
4
1 0 24408393
1 0 38848630
2 0
2 2
2 0
4
3 822201592
2 2
3 353627390
1 2 113272842
2 0
2 2
1 2 971055217
3 159495070
1 1 272088138
4
2 3
1 0 820906522
3 426088013
1 1 37651917
2 0
4
4
2 2
2 1
2 0
3 350757369
4
2 2
1 2 984584317
2 0
3 609419086
2 2
4
4
4
2 1
1 0 613852097
3 512752527
2 1
3 566201668
3 569146579
2 3
1 0 427007737
2 0
1 2 438697772
2 2
2 0
2 2
4
1 0 208855193
2 2
1 0 994938340
1 3 58410438
4
2 3
1 3 708395466
3 452576169
3 998734953
3 719338148
1 3 705704987
1 2 808440141
1 0 978443124
1 2 506572379
1 2 380698831
3 234929196
3 207501415
3 280203298
2 3
2 1
2 1
3 531258894
3 242815543
2 3
2 3
2 1
1 2 94507323
1 3 364435648
1 0 919017512
4
2 3
3 556878538
3 842168206
2 1
4
2 2
2 0
3 762178830
1 0 77772185
4
2 3
1 0 282003665
2 0
1 3 755044926
2 3
3 173239041
2 1
2 3
2 1
3 184672618
4
2 2
2 1
2 2
4
2 0
2 2
3 347488457
2 0
1 0 574697201
1 2 539229577
2 2
4
3 504589327
3 242052775
4
4
2 0
4
2 2
1 0 833292392
1 1 201335661
3 873107288
4
3 135294225
1 1 350912775
4
3 542026798
2 1
1 0 926614526
2 3
3 45886447
3 640415559
3 561796262
1 3 459934997
1 2 281598880
2 1
2 3
4
2 2
1 0 821198668
3 264518022
2 0
3 741605222
1 0 363712180
4
3 428394383
2 2
2 0
3 122200856
1 3 96830810
1 1 838525998
2 0
4
4
3 456034444
2 0
1 3 77079273
2 2
2 0
3 79553773
3 801816562
2 2
3 241759426
4
4
4
4
4
1 0 880092790
3 333889097
3 991190807
2 3
3 3384705
2 0
2 3
3 91877182
4
3 36542810
4
4
3 126457114
2 1
1 0 557956571
3 428411289
3 817738153
2 0
2 2
4
3 457295096
1 0 85857135
3 586451066
1 0 69159196
1 3 232567800
2 0
2 2
2 0
3 909750328
4
2 2
3 75592464
2 0
2 2
4
3 302523425
1 3 751427023
2 3
4
4
4
1 3 268852748
4
4
2 3
3 682833332
3 935330272
3 976139281
3 505575238
2 1
4
3 876311259
4
2 3
1 3 961691974
2 3
4
2 1
4
1 1 915409603
2 1
3 913766514
2 1
4
1 0 431387217
3 25406074
3 769310567
4
2 0
1 2 803438279
2 2
4
3 1980425
4
2 0
1 2 259106310
3 910513389
1 2 703020245
4
1 2 387676663
1 3 850762113
3 829607037
1 0 463872048
1 0 222223997
2 0
4
4
2 2
3 675669934
3 78469376
4
4
1 2 91356217
4
3 984868197
3 358985364
1 1 322429073
3 554727821
4
4
3 353977464
3 341784744
4
4
2 2
1 1 885483600
4
3 789045837
4
1 0 267036744
4
3 340139815
1 3 985699574
1 0 659685379
3 961583212
1 1 695030364
3 317565605
1 0 554879871
3 730697609
1 1 434654641
1 1 161215388
3 374421775
3 218474466
2 2
2 1
3 400512818
3 543506268
2 1
3 699302008
4
1 1 643406271
1 0 42132349
1 3 443477462
3 891938651
4
4
4
1 3 417219617
1 0 368581131
1 0 92504093
4
2 1
1 2 650619541
1 2 240565094
2 3
4
3 50125682
2 3
1 3 229322455
4
3 991179133
1 2 617655073
1 1 18062656
2 0
1 3 127527411
2 2
1 2 425175894
3 622347436
4
1 2 393449060
2 2
3 148235365
4
2 1
2 0
2 2
4
3 913241458
1 3 315705911
4
3 943172661
3 306166351
1 3 672340999
4
1 1 91000269
2 0
2 2
4
2 3
1 0 406814499
3 343383055
4
4
1 0 947994313
2 3
3 135487561
1 3 264971715
3 719937672
3 486213380
2 1
2 0
3 264389491
1 0 433779643
4
2 2
3 288247697
3 279309361
4
3 575893646
1 3 462132165
1 0 249784845
1 0 88957506
1 1 797483468
3 75697834
2 1
4
2 1
4
2 3
2 3
2 1
3 759229861
4
4
1 2 232168779
1 1 211498493
2 1
3 356522933
4
3 908632394
2 3
2 0
4
2 2
1 1 472817156
1 2 202325524
1 2 871050244
3 925981698
3 975714839
4
1 1 9745001
1 2 877704894
2 3
3 614533479
3 533598144
3 934851295
3 424832728
4
4
4
3 94914843
2 3
2 3
2 1
1 1 434972303
1 3 483073516
4
3 540308300
3 6078929
3 962038951
3 557737582
4
2 1
1 0 366654694
3 190247225
3 244407063
2 0
4
3 10795269
1 3 419734413
4
1 3 212367851
2 2
3 708755631
2 0
4
3 456963790
2 1
2 0
1 2 507896285
2 3
1 2 861389304
3 333084344
4
3 790416684
1 3 721733945
3 571862210
4
3 544256583
1 0 845523047
3 820731077
2 2
2 0
4
4
1 3 715727976
2 0
4
3 397015410
3 599174511
3 668738719
2 2
4
4
4
3 935563734
4
2 2
1 3 912560499
2 3
1 2 510191692
3 2724994
4
3 592603477
3 756337684
3 845651662
4
1 0 910237248
2 1
1 0 959458397
4
3 44814569
4
2 0
3 82111311
4
3 96780495
4
2 0
4
1 1 853432669
1 3 570721085
3 992147214
3 848765355
3 732802268
4
2 2
4
1 2 482891940
3 665470339
1 2 591240958
2 3
3 154532454
1 0 576856007
4
4
1 2 561341922
4
3 813037462
1 2 761690888
2 2
2 0
4
2 0
3 799012358
2 2
4
4
3 802490102
1 3 663642498
4
4
4
2 0
3 831938506
1 2 757727484
3 649537733
4
3 736857639
3 605016472
4
1 0 880647253
2 0
2 2
2 2
1 0 809253472
2 3
2 1
1 0 465884667
1 3 310437976
2 1
3 135597574
1 0 335691561
2 0
2 3
3 565444951
2 1
4
3 173509028
4
3 473226651
1 0 967400496
1 2 572408423
3 596167856
1 1 877641347
2 1
1 1 482664579
2 1
2 3
4
4
4
3 885692905
4
3 28042820
2 3
2 1
4
2 3
4
1 2 153679136
4
4
1 3 777181030
3 855565797
2 2
3 783844801
3 34002424
3 120307045
3 653970113
4
1 3 485673418
3 72214664
3 648336107
2 0
2 1
2 3
3 654459342
3 531030557
3 168216871
3 931973366
2 0
4
1 0 628294124
2 2
3 250501258
1 1 382223041
3 438142345
3 794080441
2 3
3 593186788
2 1
2 0
1 2 449099976
3 164320418
4
4
2 0
4
1 2 190568419
1 2 870487365
3 432761285
3 152827892
1 2 422258872
3 855858199
1 1 387172073
2 2
4
3 582585051
2 0
1 2 305535350
2 2
4
3 614227224
4
2 2
3 760467465
1 1 164939444
3 81874150
3 943729987
1 3 552835185
2 3
3 826999922
3 72431637
4
2 1
1 2 238547294
2 1
1 2 85382229
4
1 2 10193795
1 1 140043714
1 1 373920059
1 1 783275560
1 0 856264905
4
4
3 566394873
3 881775608
3 275932543
3 843438987
3 718374430
3 936696897
2 1
3 607206211
4
1 1 53858316
3 546684865
2 3
1 3 948348312
1 2 257117015
4
4
4
2 0
2 2
1 2 390006399
1 0 917810532
1 1 434404054
4
1 1 784891117
3 527842988
1 1 454562205
3 59580960
4
1 3 364524069
2 0
1 1 721907680
4
2 2
3 969254960
1 3 204276924
3 557641796
4
1 3 488703852
1 3 150637491
3 473704705
2 2
2 0
4
4
3 36288766
3 801157487
1 0 536208719
1 0 523220705
4
4
2 1
1 0 985781907
3 634213424
2 1
1 2 912123447
3 333691663
2 2
4
2 0
3 241566806
1 3 366486604
2 3
1 2 972550466
1 2 118472339
4
4
4
1 1 28012917
3 404546293
2 3
2 0
1 1 476053331
4
2 2
3 570381483
2 3
2 1
4
3 990483285
3 412482402
4
2 0
