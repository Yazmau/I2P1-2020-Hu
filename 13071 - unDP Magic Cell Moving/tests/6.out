Invalid
232566849
232566849
414463421
414463421
Invalid
Invalid
Invalid
414463421
414463421
Invalid
414463421
414463421
261330565
261330565
261330565
Invalid
Invalid
539041769
539041769
Invalid
Invalid
Invalid
765076369
765076369
765076369
765076369
765076369
765076369
646399857
261330565
Invalid
261330565
261330565
Invalid
Invalid
Invalid
Invalid
603740891
152632316
Invalid
152632316
152632316
604953294
Invalid
Invalid
806359821
806359821
806359821
261330565
261330565
Invalid
Invalid
Invalid
528042240
Invalid
353192088
806359821
649471902
Invalid
Invalid
604953294
604953294
604953294
Invalid
467250912
467250912
289738367
807613404
807613404
Invalid
788758322
424008614
Invalid
Invalid
646722049
646722049
Invalid
69876497
557419813
Invalid
Invalid
981930636
516925237
516925237
Invalid
Invalid
Invalid
Invalid
Invalid
833311783
Invalid
900863891
833311783
833311783
900863891
Invalid
749062786
593096016
Invalid
Invalid
709763771
Invalid
Invalid
Invalid
Invalid
807613404
Invalid
Invalid
245750888
201046694
894420438
Invalid
Invalid
843882482
917958945
917958945
Invalid
24408393
Invalid
Invalid
Invalid
159495070
Invalid
820906522
820906522
350757369
Invalid
159495070
159495070
159495070
Invalid
Invalid
Invalid
512752527
Invalid
Invalid
118210004
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
280203298
280203298
Invalid
762178830
184672618
808440141
Invalid
Invalid
347488457
242052775
242052775
762178830
Invalid
873107288
Invalid
135294225
561796262
Invalid
Invalid
741605222
Invalid
Invalid
762178830
762178830
241759426
241759426
241759426
241759426
241759426
Invalid
91877182
36542810
36542810
817738153
Invalid
Invalid
Invalid
909750328
75592464
Invalid
126457114
126457114
126457114
126457114
126457114
126457114
876311259
961691974
505575238
Invalid
302523425
Invalid
769310567
Invalid
769310567
1980425
Invalid
Invalid
910513389
Invalid
Invalid
463872048
463872048
78469376
78469376
Invalid
78469376
554727821
554727821
341784744
341784744
1980425
789045837
Invalid
789045837
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
699302008
Invalid
Invalid
891938651
891938651
891938651
Invalid
Invalid
Invalid
891938651
Invalid
891938651
Invalid
543506268
Invalid
Invalid
Invalid
Invalid
622347436
Invalid
148235365
201335661
Invalid
913241458
Invalid
306166351
306166351
Invalid
343383055
343383055
Invalid
Invalid
264389491
279309361
Invalid
Invalid
Invalid
Invalid
306166351
91000269
759229861
759229861
Invalid
356522933
926614526
Invalid
Invalid
Invalid
975714839
Invalid
Invalid
424832728
424832728
424832728
Invalid
Invalid
486213380
557737582
Invalid
264389491
Invalid
10795269
Invalid
10795269
Invalid
Invalid
333084344
Invalid
571862210
Invalid
820731077
820731077
Invalid
609419086
820731077
820731077
820731077
935563734
2724994
845651662
Invalid
968170837
44814569
82111311
96780495
668738719
732802268
96780495
Invalid
Invalid
Invalid
154532454
154532454
154532454
Invalid
813037462
813037462
813037462
802490102
802490102
802490102
Invalid
649537733
605016472
Invalid
Invalid
Invalid
Invalid
732802268
173509028
Invalid
Invalid
853432669
853432669
853432669
885692905
28042820
596167856
Invalid
596167856
596167856
Invalid
653970113
Invalid
967400496
Invalid
Invalid
164320418
164320418
628294124
Invalid
Invalid
Invalid
164320418
Invalid
582585051
614227224
Invalid
Invalid
72431637
Invalid
28042820
Invalid
Invalid
Invalid
Invalid
28042820
28042820
607206211
Invalid
Invalid
936696897
936696897
936696897
Invalid
Invalid
Invalid
936696897
Invalid
Invalid
59580960
Invalid
856264905
Invalid
557641796
Invalid
Invalid
473704705
473704705
Invalid
Invalid
801157487
801157487
912123447
Invalid
Invalid
634213424
634213424
634213424
Invalid
Invalid
856264905
570381483
412482402
